config/feature.js

```diff
@@ -23,6 +23,10 @@ function isFirefoxPanel() {
   return DebuggerTarget == "firefox-proxy";
 }
 
+function isFirefox() {
+  return /firefox/i.test(navigator.userAgent)
```

Comment:
perhaps. but i think our usecase if fairly limited. honestly, i wouldn't be surprised if we improve this later